package-lock.json

```diff
@@ -55,6 +55,18 @@
         "recompose": "0.28.0 - 0.30.0"
       }
     },
+    "@material-ui/lab": {
```

Comment:
what is material UI lab?

Reply:
I don't know... when I run the program, it gave me the error msg and I had to install the lab